src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.cs

```diff
@@ -1165,7 +1165,7 @@ internal IEnumerable<SecurityKey> GetContentEncryptionKeys(JsonWebToken jwtToken
             {
                 try
                 {
-#if NET472 || NET6_0
+#if NET472 || NET6_0 || NET8_0
```

Comment:
Can't we use `NET6_0_OR_GREATER`.
See the table in https://learn.microsoft.com/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation

Reply:
Good suggestion. Done.